app/components/header.tsx

```diff
@@ -32,24 +31,36 @@ export default function Header() {
       shadow="2xl"
       fontSize="18"
       paddingY="2"
-      paddingX={{ base: '1', md: '10' }}
+      paddingX={{ base: '1', md: '5' }}
       justifyContent="space-between"
+      position="absolute"
     >
       <Hide below="lg">
-        <HStack color="white" justifyContent="flex-start">
-          <Link to={{ pathname: '/domains' }}>
-            <Flex alignItems="center">
-              <TriangleUpIcon marginRight="2" />
-              <Text>Domains</Text>
-            </Flex>
-          </Link>
+        <HStack color="white" justifyContent="flex-start" gap="10">
+          <Flex justifySelf="flex-start">
+            <IconButton
+              as={Link}
+              to={{ pathname: '/' }}
+              aria-label="Redirect to main"
+              _hover={{ backgroundColor: 'transparent' }}
+              icon={<LockIcon color="gray.600" />}
```

Comment:
Instead of adding another icon as a link to home, we could keep `My.Custom.Domain` in the header, and add a link to that. So clicking on it would take you to `/` instead

Reply:
I'm refraining from using text on header unless it's on the edges is due to alignment issues. I noticed that it never stays in the middle on different scale

Reply:
Yeah, but this will also remove it from the other pages. I think we can create another issue to fix the alignment for the header for different viewports